### ISSUE TITLE
Keep only visitors count

### ISSUE THREAD

**Issue by ebanDev** [allinurl/goaccess#2246]
Hey, I'd like to have an option to keep only unique visitors count in the view charts/table :3 
Thanks !

**Comment by allinurl** [comment]
I'm not sure I follow, you can mouse over right now the label under the chart and it will highlight that metric. Let me know if that's what you are after. Thanks

**Comment by ebanDev** [comment]
Yeah, but I'd like to have an option to remove the view count. As my Goaccess dashboard is shared with others, I'd like to keep only the visitors count cause having two numbers are a bit confusing

**Comment by allinurl** [comment]
Thanks for the feedback. Do you just need to remove it from the chart or the column as well?

**Comment by ebanDev** [comment]
I need to remove them from chart and columns
